apps/entropy-tester/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "@cprussin/tsconfig/nextjs.json",
+  "include": ["svg.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
```

Comment:
You should remove the first and last option here give you aren't in a next app so there's no `.next` folder at all, and you have no svgs so the first one doesn't make sense